Add Featured Artist Track Updates: Winter 2023 news post (#8566)
ppy/osu-wiki: news/2023-01-04-featured-artist-track-updates-winter-2023.md

---
layout: post
title: "Featured Artist Track Updates: Winter 2023"
date: 2023-01-04 13:00:00 +0000
---

Your New Year's resolution is to learn how to map, right? We've picked up over **60** new songs from **18** of our Featured Artists for you to try out!

![](/wiki/shared/news/banners/featured-artist.jpg)

> We announce new Featured Artists twice a week. Artists never stop producing music though, so we sometimes pick up songs from our backlog of artists. This post is a compilation of every new song we've licensed since [our batch last Summer](https://osu.ppy.sh/home/news/2022-07-02-featured-artist-track-updates-summer-2022).

Reviewer: ```suggestion
We announce new Featured Artists twice a week. Artists never stop producing music though, so we sometimes pick up songs from our backlog of artists. This post is a compilation of every new song we've licensed since [our batch last summer](https://osu.ppy.sh/home/news/2022-07-02-featured-artist-track-updates-summer-2022).
```

> Here's how this works: **click on an artist's name or banner to visit their Featured Artist page.** All new songs are marked with a slick *NEW* icon, which contractually obligates you to map them.

Reviewer: ```suggestion
Here's how this works: **click on an artist's name or banner to visit their Featured Artist page**. All new songs are marked with a slick *NEW* icon, which contractually obligates you to map them.
```

![](/wiki/shared/news/2022-01-02-featured-artist-track-updates-winter-2022/new.png)

> New songs are also listed below along with any relevant ranked/loved maps and an audio preview or two. These should give you a quick impression of what's in store, but also remember that all song previews are available from each Featured Artist page!

Reviewer: ```suggestion
New songs are also listed below along with any relevant Ranked/Loved maps and an audio preview or two. These should give you a quick impression of what's in store, but also remember that all song previews are available from each Featured Artist page!
```

</audio>

## [BlackY](https://osu.ppy.sh/beatmaps/artists/150)

[![](https://assets.ppy.sh/artists/150/header.jpg)](https://osu.ppy.sh/beatmaps/artists/150)

A listing of 100 songs wasn't enough for **BlackY**. We've gathered a few stragglers, including a few that you can find on [**Risa Yuzuki**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/211) too!

- **BlackY - AlterLuna**
- **BlackY - Aqua Fairy**
- **BlackY - Evolverlast**
- **BlackY - Fallin' 2 Me**
- **BlackY - Flaming Eternity**
- **BlackY - Mebius**
- **BlackY - Swing Edge**

*...and more coming very soon!*

Taste what's in store:

### BlackY - AlterLuna

<audio controls>
    <source src="https://assets.ppy.sh/artists/150/Songs/BlackY%20-%20AlterLuna.mp3" type="audio/mpeg">
</audio>

### BlackY - Mebius

<audio controls>
    <source src="https://assets.ppy.sh/artists/150/Songs/BlackY%20-%20Mebius.mp3" type="audio/mpeg">
</audio>

## [DJ Raisei](https://osu.ppy.sh/beatmaps/artists/204)

[![](https://assets.ppy.sh/artists/204/header.jpg)](https://osu.ppy.sh/beatmaps/artists/204)

> This year's osu! World Cup received a lot of love from our Featured Artists, like **DJ Raisei** and his new osu!original song *T.R.A.P.*!

Reviewer: ```suggestion
This year's osu! World Cup received a lot of love from our Featured Artists, like **DJ Raisei** and his new osu! original song *T.R.A.P.*!
```

### DJ Raisei - T.R.A.P.

Check out [this beatmap](https://osu.ppy.sh/beatmapsets/1889105) hosted by [Halgoh](https://osu.ppy.sh/users/4109923) and used in OWC's Grand Finals NM4 slot.

<audio controls>
    <source src="https://assets.ppy.sh/artists/204/Songs/DJ%20Raisei%20-%20T.R.A.P..mp3" type="audio/mpeg">
</audio>

## [Good Kid](https://osu.ppy.sh/beatmaps/artists/240)

[![](https://assets.ppy.sh/artists/240/header.jpg)](https://osu.ppy.sh/beatmaps/artists/240)

**Good Kid** makes good music. It hasn't been long since their release on osu!, so they've only had one new song released, and we've added it to their listing!

### Good Kid - No Time to Explain

Check out [this map](https://osu.ppy.sh/beatmapsets/1846926) hosted by [wafer](https://osu.ppy.sh/users/9416836)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/240/Songs/Good%20Kid%20-%20No%20Time%20to%20Explain.mp3" type="audio/mpeg">
</audio>

## [II-L](https://osu.ppy.sh/beatmaps/artists/172)

[![](https://assets.ppy.sh/artists/172/header.jpg)](https://osu.ppy.sh/beatmaps/artists/172)

> **II-L**'s songs dive into rhythmic territory that no other musicians dare to explore. With complexity like this, you'd never believe that **II-L**'s music could have vocals. In their osu!original for the recent osu! World Cup, *THE EARTH* shows that it's... somehow possible!

Reviewer: ```suggestion
**II-L**'s songs dive into rhythmic territory that no other musicians dare to explore. With complexity like this, you'd never believe that **II-L**'s music could have vocals. In their osu! original for the recent osu! World Cup, *THE EARTH* shows that it's... somehow possible!
```

### II-L feat. amelie xoxo - THE EARTH

Jump into [this map used as OWC's Grand Finals NM5](https://osu.ppy.sh/beatmapsets/1889101) hosted by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/172/Songs/II-L%20feat.%20amelie%20xoxo%20-%20THE%20EARTH.mp3" type="audio/mpeg">
</audio>

## [Imperial Circus Dead Decadence](https://osu.ppy.sh/beatmaps/artists/35)

[![](https://assets.ppy.sh/artists/35/header.jpg)](https://osu.ppy.sh/beatmaps/artists/35)

> If there's anything that the osu! audience loves more than a certain three digit number, it's **Imperial Circus Dead Decadence**. This Japanese death metal band has made an astounding impact on the osu! community thanks to songs like [Uta](https://osu.ppy.sh/beatmapsets/410162) and [Yomi yori](https://osu.ppy.sh/beatmapsets/575330), and we're here to slam their newest album down your ear-holes. In classic ICDD fashion, that album's name is *MOGARI - Shi e Fukeru Omoi wa Rikujoku Sura Kurai, Kanata no Sei o Aisuru Tame ni Inoti o Tataeru*.

Reviewer: ```suggestion
If there's anything that the osu! audience loves more than a certain three-digit number, it's **Imperial Circus Dead Decadence**. This Japanese death metal band has made an astounding impact on the osu! community thanks to songs like [Uta](https://osu.ppy.sh/beatmapsets/410162) and [Yomi yori](https://osu.ppy.sh/beatmapsets/575330), and we're here to slam their newest album down your ear-holes. In classic ICDD fashion (i.e. very long), that album's name is *MOGARI - Shi e Fukeru Omoi wa Rikujoku Sura Kurai, Kanata no Sei o Aisuru Tame ni Inoti o Tataeru*.
```

- **Imperial Circus Dead Decadence - Bunretsu Shita Doke to Hakkyo no Shudo Onna**
- **Imperial Circus Dead Decadence - Goku**
- **Imperial Circus Dead Decadence - Hitsuunaru Kyouon wa Kanashiki Kaikou mode:Alpha**
- **Imperial Circus Dead Decadence - Hitsuunaru Kyouon wa Kanashiki Kaikou modeOmega**
- [**Imperial Circus Dead Decadence - Hyakki Yakou -Pandemonic Night Parade-**](https://osu.ppy.sh/beatmapsets/1776682) hosted by [Hantrik](https://osu.ppy.sh/users/10059654)
- **Imperial Circus Dead Decadence - Kuroki Sakura wa Ai Sono Shi o Kou**
- [**Imperial Circus Dead Decadence - Misogiharae no Kamiumi wa Notamai, Magabarai no Shokuzai wa Chikau.**](https://osu.ppy.sh/beatmapsets/1781688) hosted by [Camo](https://osu.ppy.sh/users/5194391)
- **Imperial Circus Dead Decadence - Shinbatsu o Tadori Kyoukotsu ni Itaru (remaster)**
- **Imperial Circus Dead Decadence - Tensei**
- **Imperial Circus Dead Decadence - Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo. -Mogari-**
- **Imperial Circus Dead Decadence - Yoruha Tsumitobachi no Rasen**

> Here's some examples of what could the future's next 1000pp HR play:

Reviewer: ```suggestion
Here's some examples of what could the future's next 1000 pp HR play:
```

### Imperial Circus Dead Decadence - Kuroki Sakura wa Ai Sono Shi o Kou

<audio controls>
    <source src="https://assets.ppy.sh/artists/35/MOGARI%20-%20Shi%20e%20Fukeru%20Omoi%20wa%20Rikujoku%20Sura%20Kurai%2C%20Kanata%20no%20Sei%20o%20Aisuru%20Tame%20ni%20Inoti%20o%20Tataeru%2FImperial%20Circus%20Dead%20Decadence%20-%20Kuroki%20Sakura%20wa%20Ai%20Sono%20Shi%20o%20Kou.mp3" type="audio/mpeg">
</audio>

### Imperial Circus Dead Decadence - Tensei

<audio controls>
    <source src="https://assets.ppy.sh/artists/35%2FMOGARI%20-%20Shi%20e%20Fukeru%20Omoi%20wa%20Rikujoku%20Sura%20Kurai%2C%20Kanata%20no%20Sei%20o%20Aisuru%20Tame%20ni%20Inoti%20o%20Tataeru%2FImperial%20Circus%20Dead%20Decadence%20-%20Tensei.mp3" type="audio/mpeg">
</audio>

## [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58)

[![](https://assets.ppy.sh/artists/58/header.jpg)](https://osu.ppy.sh/beatmaps/artists/58)

> **Kurokotei** and **Sennzai** teamed up to create an original song for osu! World Cup's Semi Finals Tiebreaker! It even came jam-packed with background art created by fellow Featured Artist [Se-U-Ra](https://osu.ppy.sh/beatmaps/artists/112). Can musicians draw? Find out for yourself:

Reviewer: ```suggestion
**Kurokotei** and **Sennzai** teamed up to create an original song for osu! World Cup's Semifinals tiebreaker! It even came jam-packed with background art created by fellow Featured Artist [Se-U-Ra](https://osu.ppy.sh/beatmaps/artists/112). Can musicians draw? Find out for yourself:
```

### Kurokotei feat. Sennzai - escape (the looking-glass, and what alice found there)

> Escape down the rabbithole and into [this map used as OWC's Semi Finals Tiebreaker](https://osu.ppy.sh/beatmapsets/1880652) hosted by [Realazy](https://osu.ppy.sh/users/918297)!

Reviewer: ```suggestion
Escape down the rabbit hole and into [this map used as OWC's Semifinals tiebreaker](https://osu.ppy.sh/beatmapsets/1880652) hosted by [Realazy](https://osu.ppy.sh/users/918297)!
```

<audio controls>
    <source src="https://assets.ppy.sh/artists/58/Songs/Kurokotei%20feat.%20Sennzai%20-%20escape%20(the%20looking-glass%2C%20and%20what%20alice%20found%20there).mp3" type="audio/mpeg">
</audio>

## [Rabbit House](https://osu.ppy.sh/beatmaps/artists/242)

[![](https://assets.ppy.sh/artists/242/header.jpg)](https://osu.ppy.sh/beatmaps/artists/242)

**Rabbit House** (a.k.a. the dude who mapped [FREEDOM DiVE](https://osu.ppy.sh/beatmapsets/39804)) helped out with the osu! World Cup too with their song *Sanctuary*.

### Rabbit House - Sanctuary

Explore [this beatmap](https://osu.ppy.sh/beatmapsets/1889094) hosted by [funny](https://osu.ppy.sh/users/8473573) and used in OWC's Grand Finals HR4 slot.

<audio controls>
    <source src="https://assets.ppy.sh/artists/242/Songs/Rabbit%20House%20-%20Sanctuary.mp3" type="audio/mpeg">
</audio>

## [Raimukun](https://osu.ppy.sh/beatmaps/artists/247)

[![](https://assets.ppy.sh/artists/247/header.jpg)](https://osu.ppy.sh/beatmaps/artists/247)

> Another artist, another OWC osu!original. **Raimukun** showcases an endless string of piano notes in their newest tune *Myths Orbis*!

Reviewer: ```suggestion
Another artist, another OWC osu! original. **Raimukun** showcases an endless string of piano notes in their newest tune *Myths Orbis*!
```

### Raimukun - Mythis Orbis

> Muscle spasm your way through [this map used as OWC's Grand Finals NM2](https://osu.ppy.sh/beatmapsets/1889095) hosted by [knowledgeking](https://osu.ppy.sh/users/8022517)!

Reviewer: ```suggestion
Muscle-spasm your way through [this map used as OWC's Grand Finals NM2](https://osu.ppy.sh/beatmapsets/1889095) hosted by [knowledgeking](https://osu.ppy.sh/users/8022517)!
```

<audio controls>
    <source src="https://assets.ppy.sh/artists/247/Songs/Raimukun%20-%20Myths%20Orbis.mp3" type="audio/mpeg">
</audio>

## [Redside](https://osu.ppy.sh/beatmaps/artists/184)

[![](https://assets.ppy.sh/artists/184/header.jpg)](https://osu.ppy.sh/beatmaps/artists/184)

> You know the drill. osu! original here from last year's osu! World Cup. This one's from the Quarter Finals though, so it might be a tad more playable than other originals!

Reviewer: ```suggestion
You know the drill. osu! original here from last year's osu! World Cup. This one's from the Quarterfinals though, so it might be a tad more playable than other originals!
```

### Redside - Sidequest

> Experience [this map used as OWC's Quarter Finals HR3](https://osu.ppy.sh/beatmapsets/1876071) hosted by [Feiri](https://osu.ppy.sh/users/3214844)!

Reviewer: ```suggestion
Experience [this map used as OWC's Quarterfinals HR3](https://osu.ppy.sh/beatmapsets/1876071) hosted by [Feiri](https://osu.ppy.sh/users/3214844)!
```

### SAMString - Akarui Taiyo

Try out some maps from the osu!catch Mapping World Cup:

- [Team Europe's submission](https://osu.ppy.sh/beatmapsets/1897430)
- [Team Hong Kong's submission](https://osu.ppy.sh/beatmapsets/1897440)
- [Team Canada's winning submission](https://osu.ppy.sh/beatmapsets/1897433)

<audio controls>
    <source src="https://assets.ppy.sh/artists/273/Songs/SAMString%20-%20Akarui%20Taiyo.mp3" type="audio/mpeg">
</audio>

### SAMString - Crystalised Euphoria

Play through [the map from the tournament](https://osu.ppy.sh/beatmapsets/1819473) hosted by [lxcors](https://osu.ppy.sh/users/11194087)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/273/Songs/SAMString%20-%20Crystalised%20Euphoria.mp3" type="audio/mpeg">
</audio>

## [seatrus](https://osu.ppy.sh/beatmaps/artists/185)

[![](https://assets.ppy.sh/artists/185/header.jpg)](https://osu.ppy.sh/beatmaps/artists/185)

> **seatrus** joins us with two types of music: the first 8 songs are from his ever-growing backlog of tunes, and the remaining 1 song was created specifically for osu! World Cup 2022's Grand Finals Tiebreaker!

Reviewer: ```suggestion
**seatrus** joins us with two types of music: the first 8 songs are from his ever-growing backlog of tunes, and the remaining 1 song was created specifically for osu! World Cup 2022's Grand Finals tiebreaker!
```

- **Aoi feat. Sennzai - Bloody Garden**
- **Gyapuichi feat. Sennzai - Seize Your Future**
- **INFX, Ella Jung feat. Sennzai - UNINHIBITED**
- **Kasokaso (Prismagic) feat. Sennzai - solitude**
- **PRIDASK feat. Sennzai - Not a God**
- **Yuichiro Tsukagoshi (NanosizeMir) feat. Sennzai - Homura**

### INFX, Ella Jung feat. Sennzai - UNINHIBITED

<audio controls>
    <source src="https://assets.ppy.sh/artists/110/VTTRACT/INFX%2C%20Ella%20Jung%20feat.%20Sennzai%20-%20UNINHIBITED.mp3" type="audio/mpeg">
</audio>

### Gyapuichi feat. Sennzai - Seize Your Future

<audio controls>
    <source src="https://assets.ppy.sh/artists/110/VTTRACT/Gyapuichi%20feat.%20Sennzai%20-%20Seize%20Your%20Future.mp3" type="audio/mpeg">
</audio>

## [Sephid](https://osu.ppy.sh/beatmaps/artists/196)

[![](https://assets.ppy.sh/artists/196/header.jpg)](https://osu.ppy.sh/beatmaps/artists/196)

Considering **Sephid** joined osu! thanks to [a map from OWC 2020](https://osu.ppy.sh/beatmapsets/1189616), it's only appropriate that their newest tune comes straight from osu! World Cup 2022.

### Sephid - Thunderstrike 1988

> Have fun with [this map used as OWC's Quarter Finals HD3](https://osu.ppy.sh/beatmapsets/1889109) hosted by [Feiri](https://osu.ppy.sh/users/3214844)!

Reviewer: ```suggestion
Have fun with [this map used as OWC's Quarterfinals HD3](https://osu.ppy.sh/beatmapsets/1889109) hosted by [Feiri](https://osu.ppy.sh/users/3214844)!
```

<audio controls>
    <source src="https://assets.ppy.sh/artists/196/Songs/Sephid%20-%20Thunderstrike%201988.mp3" type="audio/mpeg">
</audio>

## [Street](https://osu.ppy.sh/beatmaps/artists/89)

[![](https://assets.ppy.sh/artists/89/header.jpg)](https://osu.ppy.sh/beatmaps/artists/89)

It's been over 3 years since **Street** drove into our Featured Artist collection, so we've brought their newest album *Iroha* into the mix. This oriental-styled batch of songs showcases some of **Street**'s strongest production talents, along with a re-imagined version of his classic song [Sakura Fubuki](https://osu.ppy.sh/beatmapsets/362989).

- **Street - hakai**
- **Street - hayate (with Kamiya Hiroto)**
- **Street - kamui (feat. el ma Riu)**
- **Street - reimei**
- **Street - tsumujikaze**
- **Street - Sakura Fubuki (Iroha ver.)**

### Street - hayate (with Kamiya Hiroto)

> Enjoy [this map used as OWC's Semi Finals DT4 slot](https://osu.ppy.sh/beatmapsets/1880316) hosted by [KKipalt](https://osu.ppy.sh/users/6889573)!

Reviewer: ```suggestion
Enjoy [this map used as OWC's Semifinals DT4 slot](https://osu.ppy.sh/beatmapsets/1880316) hosted by [KKipalt](https://osu.ppy.sh/users/6889573)!
```

<audio controls>
    <source src="https://assets.ppy.sh/artists/89/Iroha/Street%20-%20hayate%20(with%20Kamiya%20Hiroto).mp3" type="audio/mpeg">
</audio>

### Street - Sakura Fubuki (Iroha ver.)

<audio controls>
    <source src="https://assets.ppy.sh/artists/89/Iroha/Street%20-%20Sakura%20Fubuki%20(Iroha%20ver.).mp3" type="audio/mpeg">
</audio>

### Street - tsumujikaze

<audio controls>
    <source src="https://assets.ppy.sh/artists/89/Iroha/Street%20-%20tsumujikaze.mp3" type="audio/mpeg">
</audio>

## [Thaehan](https://osu.ppy.sh/beatmaps/artists/7)

[![](https://assets.ppy.sh/artists/7/header.jpg)](https://osu.ppy.sh/beatmaps/artists/7)

> One of our oldest Featured Artists **Thaehan** is still around! Their newest osu!original *All or Nothing* immediately slammed its way into the scene as one of last year's osu! World Cup.

Reviewer: semantically a bit off

```suggestion
One of our oldest Featured Artists **Thaehan** is still around! Their newest osu! original *All or Nothing* immediately slammed its way into the scene as one of last year's osu! World Cup originals.
```

### Thaehan - All or Nothing

> Experience [this map used as OWC's Semi Finals HR1](https://osu.ppy.sh/beatmapsets/1880625) hosted by [Zelq](https://osu.ppy.sh/users/8953955)!

Reviewer: ```suggestion
Experience [this map used as OWC's Semifinals HR1](https://osu.ppy.sh/beatmapsets/1880625) hosted by [Zelq](https://osu.ppy.sh/users/8953955)!
```
